test/test_create_entities.py

```diff
@@ -0,0 +1,135 @@
+                {
+                    "client": {
+                        "id": "client0",
+                        "uriOptions": {
+                            "retryReads": True
+                        },
+                    }
+                },
+                {
+                    "database": {
+                        "id": "database0",
+                        "client": "client0",
+                        "databaseName": "dat"
+                    }
+                },
+                {
+                    "collection": {
+                        "id": "collection0",
+                        "database": "database0",
+                        "collectionName": "dat"
+                    }
+                }
+            ],
+
+            "tests": [
+                {
+                    "description": "test loops",
+                    "operations": [
+                        {
+                            "name": "loop",
+                            "object": "testRunner",
+                            "arguments": {
+                                "storeIterationsAsEntity": "iterations",
+                                "storeSuccessesAsEntity": "successes",
+                                "storeFailuresAsEntity": "failures",
+                                "storeErrorsAsEntity": "errors",
+                                "numIterations": 100,
+                                "operations": [
+                                    {
+                                      "name": "insertOne",
+                                      "object": "collection0",
+                                      "arguments": {
+                                          "document": {
+                                              "_id": 1,
+                                              "x": 44
+                                          }
+                                      }
+
+                                    },
+                                    {
+                                        "name": "insertOne",
+                                        "object": "collection0",
+                                        "arguments": {
+                                            "document": {
+                                                "_id": 1,
+                                                "x": 44
+                                            }
+                                        }
+
+                                    }
+                                ]
+                            }
+                        }
+                    ]
+                }
+            ]
+        }
+
+        self.scenario_runner.TEST_SPEC = spec
+        self.scenario_runner.setUp()
+        self.scenario_runner.run_scenario(spec["tests"][0])
+        final_entity_map = self.scenario_runner.entity_map
+        for entity in ["errors", "failures", "successes", "iterations"]:
+            self.assertIn(entity, final_entity_map)
```

Comment:
Please do some assertions on the saved objects to make sure they are the types we expect.

Reply:
Done.